resource-managers/yarn/src/test/scala/org/apache/spark/deploy/yarn/ClientSuite.scala

```diff
@@ -106,7 +106,7 @@ class ClientSuite extends SparkFunSuite
     val conf = new Configuration()
     val sparkConf = new SparkConf()
       .set(SPARK_JARS, Seq(SPARK))
-      .set(USER_CLASS_PATH_FIRST, true)
+      .set(DRIVER_USER_CLASS_PATH_FIRST, true)
```

Comment:
To @cxzl25, FYI, usually, Apache Spark is reluctant to change a test coverage because the deprecated one is still used in the production.

Reply:
Please make a new test case for `DRIVER_USER_CLASS_PATH_FIRST`.